tests/console/sqlite3.pm

```diff
@@ -0,0 +1,61 @@
+# SUSE's openQA tests
+#
+# Copyright (C) 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+# Summary: Test sqlite3 package
+# Maintainer: Tina Müller <tina.mueller@suse.com>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils 'zypper_call';
+use Data::Dumper;
```

Comment:
I guess that is a leftover from debugging.

Reply:
yep, I always forget to take that out and since it's core, it's usually not a problem.
should I remove it?

Reply:
Yes, remove it. I often enough forget it too :-)